src/toil/lib/web.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2024 Regents of the University of California
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Contains functions for making web requests with Toil.
+
+All web requests should go through this module, to make sure they use the right
+user agent.
+
+>>> from toil.lib.web import session
```

Comment:
Since we already have a `session.py` in Toil that we import, could we call this something else?  Maybe `requests_w_toil_headers` since this is a drop-in replacement for the `requests` import?